application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -46,14 +46,26 @@ Analytics.Code.AnalyticsDashboardClass_endDate.hint=End time of the date interva
 Analytics.Code.AnalyticsDashboardClass_description.hint=Additional information to be displayed.
 Analytics.Code.AnalyticsDashboardClass_DisplayedStatistics.hint=List of statistics widgets that will be displayed in the current dashboard, in order.
 
-# Live table keys
+## Config
+Analytics.Code.ConfigurationClass_trackingCode=Tracking Code
+Analytics.Code.ConfigurationClass_idSite=Id Site
+Analytics.Code.ConfigurationClass_requestAddress=Request Address
+Analytics.Code.ConfigurationClass_authToken= Authentication Token
+Analytics.Code.ConfigurationClass_trackingCode.hint=Tracking code received from Matomo. The 'u' variable might be different than the one that is initially generated by Matomo if the server you want to track is on another machine. To fix this, go to the integration tab, copy the Matomo URL, and replace the value of 'u' with the Matomo URL.
+Analytics.Code.ConfigurationClass_idSite.hint=The ID of the site that you want to see the statistics for (found in the integration tab).
+Analytics.Code.ConfigurationClass_requestAddress.hint=The Matomo url (found in the integration tab).
+Analytics.Code.ConfigurationClass_authToken.hint=Token generated by Matomo to be capable of using the API to view the data.
+## Live table keys
 analytics.livetable.doc.title=Page Title
 analytics.livetable.startDate = Start Date
 analytics.livetable.endDate = End Date
 analytics.livetable.doc.name=Page Name
 analytics.livetable.emptyvalue=Not Defined
 
-# Others
+## Admin Section
+admin.analytics.application=Analytics
```

Comment:
You shoudl merge this with the `Config` section above, under the `Admin section` title